src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let temp = 0;
```

Comment:
why do you need temp? just use sum

Reply:
I cant understand how at the same time i must return sum and set sum = 0 to clear amount...